packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -539,28 +668,49 @@ contract OPContractsManager is ISemver {
         return abi.encodeCall(IDelayedWETH.initialize, (_input.roles.opChainProxyAdminOwner, superchainConfig));
     }
 
-    function encodePermissionedDisputeGameConstructor(
-        DeployInput memory _input,
-        DeployOutput memory _output
+    function encodePermissionlessFDGConstructor(IFaultDisputeGame.GameConstructorParams memory _params)
+        internal
+        view
+        virtual
+        returns (bytes memory)
+    {
+        return abi.encode(
+            _params.gameType,
+            _params.absolutePrestate,
+            _params.maxGameDepth,
+            _params.splitDepth,
+            _params.clockExtension,
+            _params.maxClockDuration,
+            _params.vm,
+            _params.weth,
+            _params.anchorStateRegistry,
+            _params.l2ChainId
+        );
+    }
+
+    function encodePermissionedFDGConstructor(
+        IFaultDisputeGame.GameConstructorParams memory _params,
+        address _proposer,
+        address _challenger
     )
         internal
         view
         virtual
         returns (bytes memory)
     {
         return abi.encode(
```

Comment:
Similarly `abi.encodeCall(IPermissionedDisputeGame.__constructor__, (_params, _proposer, _challenger))`

Reply:
See above - I had issues getting this to work.